mongoDB/mongo.manager.go

```diff
@@ -1,19 +1,31 @@
-package mongoManager
+package mongomanager
 
 import (
 	"context"
 	"fmt"
 	"log"
+	"os"
 
+	"github.com/joho/godotenv"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+//CNX is a mongo client
 var CNX = Connection()
 
+//Connection is a function that creates a mongo client and
+//connects it to the db and pings/checks the connection
 func Connection() *mongo.Client {
+
+	// loading the dotenv parameters
+	envErr := godotenv.Load()
+	if envErr != nil {
+		log.Fatal("Error loading .env file")
+	}
+
 	// Set client options
-	clientOptions := options.Client().ApplyURI("mongodb://root:example@0.0.0.0:27017")
+	clientOptions := options.Client().ApplyURI(os.Getenv("MONGO_URL"))
```

Comment:
Not having a default value to the envars is bad practice, use Viper

Reply:
used viper, done